src/Makefile

```diff
@@ -86,6 +86,19 @@ DISABLE_SYSC?=no
 DISABLE_CISCAT?=no
 DISABLE_STRIP_SYMBOLS?=no
 
+ifneq (${TARGET}, winagent)
+ifeq (${uname_S}, Linux)
+EXECUTABLE_FILES := $(shell find . -maxdepth 1 -type f ! -name "*.so"  -perm /a=x)
+EXECUTABLE_FILES += $(shell find . -type f -not -path "./external/*" -name "*.so" -perm /a=x)
+EXECUTABLE_FILES:= $(EXECUTABLE_FILES:./%=%)
+endif
+ifeq (${uname_S}, Darwin)
+EXECUTABLE_FILES=$(shell find . -maxdepth 1 -type f -not -path "./external/*"  -exec /bin/sh -c "file {} | grep 'bit executable' > /dev/null" \; -print)
```

Comment:
Hi @RamiroRD 

The option `-not -path "./external/*"` isn't necessary because the `maxdepth` option already limits the recursive operation of the find command. Also, I think we'll have some duplicate .so's. Am I right?

Possible duplicate .so executables

- libwazuhhex.so
- libwazuhshared.so

Reply:
True. We shouldn't need the path exclusion. However I'm not sure what you mean by those duplicates.